src/tests/App.test.tsx

```diff
@@ -3,13 +3,14 @@ import { render, screen } from "@testing-library/react";
 import App from "../App";
 
 describe("App", () => {
-  it("Renders Hello Menternshift!", () => {
+  it("Renders H1", () => {
     render(<App />);
 
     expect(
       screen.getByRole("heading", {
         level: 1,
+        name: /Experience is just a project away/,
```

Comment:
I don't expect this title to change anytime soon as it maybe a good idea to consider 
```suggestion
        name: /experience is just a project away/i,
```
addition of /i to allow for case insensitive. This title is not expected to change but for other future text, might be a good idea to add.